scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua

```diff
@@ -5,17 +5,20 @@
 
 function onMobFight(mob,target)
     if (mob:getHPP() <= 50 and mob:getLocalVar("Saplings") < 1) then
-        SpawnMob(mob:getID()+1):updateEnmity(target);
-        SpawnMob(mob:getID()+2):updateEnmity(target);
-        SpawnMob(mob:getID()+3):updateEnmity(target);
-        SpawnMob(mob:getID()+4):updateEnmity(target);
-        SpawnMob(mob:getID()+5):updateEnmity(target);
-        mob:setLocalVar("Saplings", 1);
+        SpawnMob(mob:getID()+1):updateEnmity(target)
+        SpawnMob(mob:getID()+2):updateEnmity(target)
+        SpawnMob(mob:getID()+3):updateEnmity(target)
+        SpawnMob(mob:getID()+4):updateEnmity(target)
+        SpawnMob(mob:getID()+5):updateEnmity(target)
+        mob:setLocalVar("Saplings", 1)
     end
-end;
+end
 
 function onMobDeath(mob, player, isKiller)
-    local JACK = mob:getID()+6;
-    SpawnMob(JACK):updateEnmity(player);
+end
+
+function onMobDespawn(mob)
+    local JACK = mob:getID()+6
+    SpawnMob(JACK)
     GetMobByID(JACK):setPos( mob:getXPos(), mob:getYPos(), mob:getZPos(), 0);
```

Comment:
setSpawn, use it or lose it sister.

Reply:
breaks behavior of him walking back up the cliff if pulled down so I wasn't sure I should make that change (I actually did then reverted it 2nd guessing myself). to get him 100% retail he should anchor to to where willow died *except* when willow dies down the cliff side at which point he goes back to the area where willow spawned at.

Still want me to switch it?

Reply:
leave a comment in the script for now with retail behaviour vs current

Reply:
done